sessions/recap-project-5/example-implementation/pages/favorites.js

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+import ArtPiecesPreview from "../components/ArtPiecesPreview";
+
+const ImageGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(1fr);
+  grid-gap: 5px;
+  width: 600px;
+  height: 100%;
+`;
+
+export default function FavoritePieces({
+  pieces,
+  handleClickToggleFavorite,
+  artPiecesInfo,
+}) {
```

Comment:
According to user story 6, `favorites.js` should render the component `ArtPieces`.